### PR TITLE
2.9.1 release

### PR DESCRIPTION
**Pull request by dvntucker**
Creating the 2.9.1 release website updates.

### CODE REVIEW

File: release/2.9.1/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: release_29
+title: GeoServer
+version: 2.9.1
+jira_version: 13001
+release_date: August 4th, 2016
+announce: 2016/05/30/geoserver-2-9-0-released/
```
Comment (jodygarnett) [inline release/2.9.1/index.html]: is this going to be the correct url?

**Comment by jodygarnett** [comment]
This looks good, we should merge just before blog post.

File: release/2.9.1/index.html (hunk 1 above)
Comment (bencaradocdavies) [inline release/2.9.1/index.html]: @jodygarnett this is never going to be the right url as it has the wrong number and date. This is the URL for 2.9.0.

File: release/stable/index.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: release_29
 title: GeoServer
-version: 2.9.0
-jira_version: 13003
-release_date: May 30, 2016
+version: 2.9.1
+jira_version: 13001
+release_date: August 4th, 2016
 announce: 2016/05/30/geoserver-2-9-0-released/
```
Comment (bencaradocdavies) [inline release/stable/index.html]: This one too. Just come back and fix them when the blog is posted. No worries!

**Comment by dvntucker** [comment]
I'll update those now.

**Comment by dvntucker** [comment]
Announcement updated https://github.com/geoserver/geoserver.github.io/pull/36

**Pull request merged**